app/models/deploy.rb

```diff
@@ -201,6 +204,13 @@ def csv_line
     ]
   end
 
+  def as_json(options = {})
+    raise unless options.empty?
+    hash = super(methods: [:status, :url, :production])
+    hash["summary"] = summary_for_timeline
+    hash
+  end
```

Comment:
same as the `DeploySerializer` it replaces ... DeploySerializer will be removed later